app/models/bgeigie_import.rb

```diff
@@ -88,6 +88,24 @@ def finalize!
     self.update_column(:status, 'done')
   end
 
+  def fixdrive!
+    import_measurements_fix
+    update_counter_caches
+    confirm_status(:measurements_added)
+    self.update_column(:status, 'done')
```

Comment:
This format is consistent (for better or worse) with other methods in the class (finalize!, approve!)